src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -22,30 +22,63 @@ abstract class Vendedor {
     certificaciones.add(certificacion)
   }
 
-  fun esFirme() = this.puntajeCertificaciones() >= 30
+  fun esFirme() = (this.puntajeCertificaciones()) >= (30)
 
   fun certificacionesDeProducto() = certificaciones.count { it.esDeProducto }
   fun otrasCertificaciones() = certificaciones.count { !it.esDeProducto }
 
   fun puntajeCertificaciones() = certificaciones.sumBy { c -> c.puntaje }
+
+  abstract fun esVendedorInfluyente(): Boolean
+
 }
 
 // En los parámetros, es obligatorio poner el tipo
 class VendedorFijo(val ciudadOrigen: Ciudad) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudad == ciudadOrigen
   }
+
+  // si es fijo , no es influyente
+  override fun esVendedorInfluyente(): Boolean {
+    return false
+  }
+
 }
 
 // A este tipo de List no se le pueden agregar elementos una vez definida
 class Viajante(val provinciasHabilitadas: List<Provincia>) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return provinciasHabilitadas.contains(ciudad.provincia)
   }
+
+  //la población total sumando todas las provincias donde está habilitado, debe ser de 10 millones o superior
+  override fun esVendedorInfluyente(): Boolean {
+    return this.provinciasHabilitadas.sumBy{c -> c.poblacion} >= 10000000
+  }
+
+
+
 }
 
 class ComercioCorresponsal(val ciudades: List<Ciudad>) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudades.contains(ciudad)
   }
+
+  //cantidad de provincias en las que trabaja
+  fun cantProvincias(): Int  { //aca me tira el error y no entiendo bien por que si ya le declaro int
+    val provincias = mutableSetOf<Provincia>()
+    //crear coleccion nueva
+    //poner en la coleccion cada ciudad
+    //contar elementos y devolver el int
+    ciudades.map{c -> provincias.add(c.provincia) }
+    return provincias.size
+  }
```

Comment:
Tratá de declarar consultas con `=` así te evitas declarar el tipo de dato que devuelve y queda una sintaxis más limpia (aunque está muy bien que sepas que es lo que devuelve, tenés que saber). Siguiendo con el resto además de haber mucho comentario innecesario, `map()` ya devuelve una colección nueva, no es necesario que la crees vos, podés utilizar directamente esa.
```suggestion
  fun cantProvincias() = ciudades.map {it.poblacion}toSet().size
  }
```